src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -331,14 +336,23 @@ public void testBamoutProducesReasonablySizedOutput(final boolean createBamoutIn
                                "-bamout produced a bam with over 10% fewer/more reads than expected");
         }
 
+        Assert.assertTrue(vcfOutput.exists(), "No VCF output file was not created");
+
+        // validate vcfout companion files
+        final File vcfOutFileIndex = new File(vcfOutput.getAbsolutePath() + Tribble.STANDARD_INDEX_EXTENSION);
+        final File vcfOutFileMD5 = new File(vcfOutput.getAbsolutePath() + ".md5");
+        Assert.assertEquals(vcfOutFileIndex.exists(), createVCFOutIndex, "The index file argument was not honored");
+        Assert.assertEquals(vcfOutFileMD5.exists(), createVCFOutMD5, "The md5 file argument was not honored");
```

Comment:
The tests for vcf output index/md5 generation should be a separate test case, not piggy-backed onto the test for bamout. Can you refactor? Just run the HaplotypeCaller on a tiny interval (say 1000 bases), with various combinations of the indexing/md5 args.

Reply:
Done.